colabs/scikit/decision-trees.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    os.environ["WANDB_API_KEY"] = '...'
</pre><p>Are we using <code>os.environ</code> to ask  user to login in every example? I think using <code>wandb.login()</code>  should be better since it will show the https://wandb.ai/authorize link to get the api token. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Good point. I use `os.environ` as it allows for quick prototyping. Will make the changes.